pkg/network/openshift_sdn_test.go

```diff
@@ -101,7 +101,6 @@ func TestRenderOpenShiftSDN(t *testing.T) {
 		g.Expect(err).NotTo(HaveOccurred())
 
 		tweakMetaForCompare(cur)
-		g.Expect(cur).To(Equal(upd))
```

Comment:
This looks wrong? You change `cur` but then don't look at it?

Reply:
Ah yeah, good catch.